tests/wasmfs/wasmfs_getdents.c

```diff
@@ -67,42 +95,27 @@ int main() {
   getdents(fileFd, &d, sizeof(d));
   printf("Errno: %s\n\n", strerror(errno));
   assert(errno == ENOTDIR);
+  close(fileFd);
 
-  // Try opening the root directory and read its contents.
-  fd = open("/", O_RDONLY | O_DIRECTORY);
-
-  printf("------------- Reading from root Directory -------------\n");
-  print(fd);
+  close(fd);
 
   // Try opening the dev directory and read its contents.
-  fd = open("/dev", O_RDONLY | O_DIRECTORY);
   printf("------------- Reading from /dev Directory -------------\n");
-  print(fd);
+  print("/dev");
 
   // Try to advance the offset of the directory.
   // Expect that '.' will be skipped.
-  fd = open("/working", O_RDONLY | O_DIRECTORY);
-  printf("/working file position is: %lli\n", lseek(fd, 1, SEEK_SET));
-  printf("------------- Reading from /working Directory -------------\n");
-  print(fd);
+  fd = open("/root/working", O_RDONLY | O_DIRECTORY);
+  printf("/root/working file position is: %lli\n", lseek(fd, 1, SEEK_SET));
+  printf(
+    "------------- Reading one from /root/working Directory -------------\n");
+  print_one(fd);
+  close(fd);
 
   // Try to add a file to the /working directory.
-  assert(open("/working/foobar", O_CREAT, S_IRGRP) != -1);
-  printf("/working file position is: %lli\n", lseek(fd, 0, SEEK_SET));
-  printf("------------- Reading from /working Directory -------------\n");
-  print(fd);
-
-  // The musl implementation of readdir relies on getdents.
-  DIR* pDir;
-  struct dirent* pDirent;
-  pDir = opendir("/dev");
-  assert(pDir != NULL);
-
-  while ((pDirent = readdir(pDir)) != NULL) {
-    printf("pDirent->d_name: %s\n", pDirent->d_name);
-    printf("pDirent->d_off: %lld\n", pDirent->d_off);
-    printf("pDirent->d_reclen: %hu\n", pDirent->d_reclen);
-    printf("pDirent->d_type: %hhu\n\n", pDirent->d_type);
-  }
-  closedir(pDir);
```

Comment:
What happened to this part of the test?

Reply:
We're already testing that libc use of getdents works correctly via `scandir` and `readdir` does not produce deterministic ordering, so the simplest thing to do was delete this part of the test.

Reply:
I see, ok. I guess other tests will cover readdir later anyhow, so this one can stay focused.